### PR TITLE
SETI-5740: Remove one of two pbr packages from requirements

### PR DESCRIPTION
**Pull request by tomaso**
Partially reverts https://github.com/gooddata/zuul/pull/101

### CODE REVIEW

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/zuul/job/zuul-docker-tox-zuul/125/ : FAILURE in 29m 15s
- https://checklist.intgdc.com/job/compliance/job/zuul-oss-check/37/ : SUCCESS in 10s

File: requirements.txt [hunk 1]
```diff
@@ -1,5 +1,3 @@
-pbr>=1.1.0
-
```
Comment (petr-klemsinsky) [inline requirements.txt:2]: You should maybe remove the generic requirements (without version)

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/zuul/job/zuul-docker-tox-zuul/126/ : FAILURE in 35s
- https://checklist.intgdc.com/job/compliance/job/zuul-oss-check/38/ : SUCCESS in 7s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/zuul/job/zuul-docker-tox-zuul/127/ : FAILURE in 28m 11s
- https://checklist.intgdc.com/job/compliance/job/zuul-oss-check/39/ : SUCCESS in 9s

**Pull request merged**